src/app/education/education.component.css

```diff
@@ -0,0 +1,16 @@
+.education {
+  align: center;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this usage of the unknown "align" property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=css%3Aknown-properties)